pom.xml

```diff
@@ -651,7 +661,7 @@
               <argument>5m</argument>
               <argument>sh</argument>
               <argument>-c</argument>
-              <argument>until curl -sSk -o /dev/null http://${cryostat.itest.webHost}:${cryostat.itest.grafana.port}; do sleep 2; done</argument>
+              <argument>until podman ps --format '{{.Names}} {{.Status}}' | grep ${cryostat.itest.grafana.containerName} | grep -E -i '^health'; do sleep 5; done</argument>
```

Comment:
How about this?

```bash
$ podman ps --filter=health=healthy --format="{{.Names}}" | grep cryostat-itest
cryostat-itest
```

Reply:
That's probably the perfect middle-ground between our two solutions, even better than either. Nice!